### DISCUSSION TITLE
G7: sensor errors and "?CD" as blood sugar value

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#3623] [Q&A]
On sunday afternoon I started a new G7 sensor.
After the warmup my sugar level dropped very fast (I doubled checked with my blood monitor) and xDrip reported "sensor errors"; the connection dropped and the value displayed was "?CD".
In the meanwhile I had some candies so my blood sugar reached normal values; neverthless, after an hour xDrip status didn't change: still unable to reconnect and still reporting "sensor errors" and "?CD"
![Screenshot_2024-08-11-16-46-45-335_com eveningoutpost dexdrip](https://github.com/user-attachments/assets/40f11090-b56c-4ad6-baf0-ff64be6d0392)

I experienced other "sensor errors" in the past but usually they recovered within one or two reading cycles; so I suspected a sensor failure and I applied a new one, that works normally (even if it's complaining about low battery voltage... but I'll open another thread about this)

The only difference I noticed is that the faulty sensor box reports "revision 007", while the others I used in the past report "revision 005".
![IMG_20240813_170556](https://github.com/user-attachments/assets/f2eba5b9-9f8e-43d7-999b-9c70784453f7)

Not sure if I'm providing useful informations

**Comment by Navid200** [comment]
Thanks for reporting this.

If xDrip shows readings from a G7, those readings come exactly like that from G7.

If you ever decide that something maybe wrong with the sensor, I recommend trying Dexcom app just to be sure before replacing the sensor.

With respect to cd? I will see what I can do.  If I succeed, xDrip will show LOW instead.

Thanks

**Comment by jamorham** [comment]
@dagord can you specify the xDrip version you are running?

**Reply by dagord** [reply]
I'm running the last nightly: 2024.08.07
